lib/lynx/pipe/popen.rb

```diff
@@ -0,0 +1,23 @@
+require 'lynx/pipe/basic'
+require 'logger'
+
+module Lynx
+  module Pipe
+    class POpen < Basic
+      class << self
+        attr_accessor :logger
+        def logger
```

Comment:
Probably better as an instance variable like write does it. `Lynx.Pipe.Popen.new(Logger.new(STDOUT))` for instance. Also, class name does not match filename.